test/common_quantization.py

```diff
@@ -7,13 +7,22 @@
 checking quantization api and properties of resulting modules.
 """
 
+import hypothesis
 import torch
 import torch.nn.quantized as nnq
 from common_utils import TestCase
 from torch.quantization import QuantWrapper, QuantStub, DeQuantStub, \
     default_qconfig, QConfig, default_observer, default_weight_observer, \
     default_qat_qconfig
 
+# Disable deadline testing if this version of hypthesis supports it, otherwise
+# just return the original function
+def no_deadline(fn):
```

Comment:
@jamesr66a could you move this function to hypothesis_utils.py? I feel it makes more sense in that file

Reply:
You can move it if you want to

Reply:
OK